### PR TITLE
Fixed Css bugs for Skill creator and Edit page

### LINKED ISSUE #2219: CSS Bugs
**Actual Behaviour**
- There are a lot of CSS bugs in the app. Also, many elements are not responsive.
List of components - 

- [x] Auth related components
- [x] Settings page
- [ ] About pages
- [x] Details page
- [x] Skill home(BrowseSkill, etc)
- [ ] Skill creator and Edit pages
- [ ] Admin pages
- [x] Dashboard

Feel free to add more pages, if I missed.

**Expected Behaviour**
All these bugs should be fixed.

**Would you like to work on the issue?**
I, @fragm3 @harsh-jindal

### PR DESCRIPTION
**Pull request by samagragupta**
Fixes #2219 

Changes: Fixed Css bugs for Skill creator and Edit page

Demo Link: https://pr-2253-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

**Comment by samagragupta** [comment]
edit

**Comment by harsh-jindal** [comment]
@samagragupta Also do you intend to work on mobile views of skillcreator and botbuilder ?

**Comment by samagragupta** [comment]
@akshatnitd Made the changes

**Comment by samagragupta** [comment]
@harsh-jindal I can work on that.

**Comment by harsh-jindal** [comment]
@samagragupta Please fix the mobile views in this PR itself.

**Comment by harsh-jindal** [comment]
@samagragupta As I haven't seen any progress by you to fix the skill creator and edit pages in mobile view. I'll be taking up to fix the mobile view for the same.

**Comment by samagragupta** [comment]
@harsh-jindal I will do it by today.

**Comment by harsh-jindal** [comment]
@samagragupta Cool. Please always read the comments on the issue. The issue [#2219](https://github.com/fossasia/susi.ai/issues/2219) was meant to fix all the bugs and mobile views and integrate styled components.

**Comment by samagragupta** [comment]
@harsh-jindal Fixed the mobile view

**Comment by samagragupta** [comment]
Could you please explain to me the incorrect changes?

**Comment by harsh-jindal** [comment]
@samagragupta Can you provide screenshot of the mobile view fixed by you. I don't see anything fixed.

**Comment by samagragupta** [comment]
@harsh-jindal This is because the Travis is not working.

**Comment by samagragupta** [comment]
mobile

**Comment by harsh-jindal** [comment]
@samagragupta Fix this. When fixing mobile views, you need to check whether it is fixed in all mobile views or not.
Screenshot 2019-06-19 at 8 33 51 AM
Screenshot 2019-06-19 at 8 33 57 AM

**Comment by samagragupta** [comment]
@harsh-jindal Made the changes

**Comment by samagragupta** [comment]
@harsh-jindal Its working fine on localhost
Capture
Capture

**Comment by samagragupta** [comment]
Capture

**Comment by samagragupta** [comment]
@harsh-jindal Please review
Surge link working too.

File: src/components/cms/BotBuilder/BotBuilder.css [hunk 1]
```diff
@@ -296,6 +296,16 @@ input[type=file]{
 		margin-bottom: 40px;
 	}
 }
+
+@media  (max-width:1000px){
+	.botbuilder-col{
+		position: inherit !important ;
```
Comment (fragm3) [inline src/components/cms/BotBuilder/BotBuilder.css]: Avoid using `!important`

File: src/components/cms/BotBuilder/Preview/Preview.js [hunk 2]
```diff
@@ -122,6 +122,10 @@ class Preview extends Component {
         backgroundColor: botbuilderIconColor,
         backgroundImage: `url(${botAvatar})`,
       },
+      sendButton: {
+        overflowX: 'hidden',
```
Comment (fragm3) [inline src/components/cms/BotBuilder/Preview/Preview.js]: Why? Just use overflow: 'hidden'

**Comment by samagragupta** [comment]
@fragm3 Made the changes

**Comment by harsh-jindal** [comment]
@samagragupta Fix this:
Screenshot 2019-06-20 at 10 47 42 PM
Make it like this :
Screenshot 2019-06-20 at 10 47 01 PM

**Comment by samagragupta** [comment]
@harsh-jindal Made the changes
Capture

File: src/components/cms/BotBuilder/BotWizard.js [hunk 3]
```diff
@@ -629,16 +636,16 @@ class BotWizard extends React.Component {
...
               className="botbuilder-col"
               xs={12}
               md={colPreview}
               style={{
                 display: colPreview === 0 ? 'none' : 'block',
-                position: 'fixed',
-                marginLeft: '65%',
-                height: '88%',
-                marginTop: '10px',
+                // position: 'fixed',
```
Comment (fragm3) [inline src/components/cms/BotBuilder/BotWizard.js]: Remove comment

Comment (samagragupta) [reply src/components/cms/BotBuilder/BotWizard.js]: @fragm3 Done

**Comment by samagragupta** [comment]
@akshatnitd Done
Capture

**Comment by samagragupta** [comment]
@fragm3 What to fix in this.
I think it is right.
'Category and Bot Name are on the same vertical line'

**Comment by samagragupta** [comment]
@akshatnitd @harsh-jindal Please review

**Comment by harsh-jindal** [comment]
@samagragupta Shubham has asked to you fix the info icon.
Also fix this: [Commit Message text is out of the text field] 
Also check the save and cancel button UI in iphone se.
Screenshot 2019-06-22 at 12 45 15 PM

**Comment by samagragupta** [comment]
@harsh-jindal @fragm3 Done

**Comment by harsh-jindal** [comment]
@samagragupta The info icon is still not fixed. Check where you have placed it and where it is currently in the web view. Also now it is not visible in mobile view.
Check the skill creator page in ipad and ipad pro.

When you make some changes, always check whether other things are getting affected. This is to be done everywhere. Because I don't see you following these rules. You make the same mistakes again and again. How are the mentors supposed to review your PR's when we have to again and again point out the same mistakes in the PR ?

**Comment by samagragupta** [comment]
@harsh-jindal It already fixed
Capture
Capture

**Comment by samagragupta** [comment]
@akshatnitd @harsh-jindal @fragm3 
edit

**Comment by harsh-jindal** [comment]
@samagragupta In ipad and ipad pro what i meant was make the text editor width full as the preview goes below it. 
See this:
Screenshot 2019-06-23 at 10 17 36 AM

**Comment by samagragupta** [comment]
@harsh-jindal that I don't think is a problem.

**Comment by harsh-jindal** [comment]
@samagragupta 
Current (in your PR)
Screenshot 2019-06-23 at 11 33 51 AM

Make it like this:
Screenshot 2019-06-23 at 10 17 36 AM

**Comment by samagragupta** [comment]
@harsh-jindal 
Capture

**Comment by harsh-jindal** [comment]
@samagragupta Fix this (Skill Name and Language on same line)
Screenshot 2019-06-24 at 9 47 15 AM

**Comment by samagragupta** [comment]
@harsh-jindal If they are on the same line, then I don't think it is a problem.

**Comment by harsh-jindal** [comment]
@samagragupta Try uploading an image. Please try to understand the UI we are following. If it is not coming outside the card doesn't mean it is fixed.
While transitioning from web to mobile view, In the middle try to keep the UI like web or mobile. It looks very odd currently in iPad view.

**Comment by samagragupta** [comment]
@harsh-jindal Fixed that

**Comment by samagragupta** [comment]
@fragm3 Fixed the upload image issue. While I could not understand what you want for the preview tab. As you have written it is supposed to be movable, which is already implemented. 
But I have fixed the preview position to make it similar to the current version.

**Comment by samagragupta** [comment]
@akshatnitd @harsh-jindal Please review.

**Comment by harsh-jindal** [comment]
@samagragupta How is this fixed?? After asking you several times to check the whole functionality before writing it is fixed, you seem to not understand that. Why do we have to review same mistakes again and again?? If you cannot follow the coding standards take up small bug fixes, so it does not hamper the development process. This issue was to be closed 2 weeks back, but if you work like this, i don't see this being fixed anytime soon.
Screenshot 2019-06-27 at 8 16 28 AM

**Comment by samagragupta** [comment]
@harsh-jindal Its working perfectly in my localhost.

**Comment by samagragupta** [comment]
@harsh-jindal Now working in sruge link too.

**Comment by samagragupta** [comment]
@fragm3 Made the text-align to the left.

**Comment by fragm3** [comment]
@samagragupta Kindly see the screenshot and fix the alignments.

PR changes:

Screenshot 2019-06-27 at 8 54 37 PM

Currently:

Screenshot 2019-06-27 at 8 54 43 PM

**Comment by samagragupta** [comment]
@fragm3 Fixed that

**Comment by fragm3** [comment]
@akshatnitd Should I take this up in my next PR. We have waited long enough for this PR.

**Comment by samagragupta** [comment]
@fragm3 But now what is wrong in that PR. Without anything wrong, I think there is no need to close.
Else if there is something wrong, mention them. I will fix them.

**Comment by fragm3** [comment]
Fix alignment wrt to Paper, https://github.com/fossasia/susi.ai/pull/2253#pullrequestreview-254286205

**Comment by samagragupta** [comment]
@fragm3 Fixed that

**Comment by samagragupta** [comment]
@akshatnitd @harsh-jindal 
skkkkill

**Comment by samagragupta** [comment]
Conflicts are too complex.
Closing this PR. 
Will open another PR.

**Pull request closed without merging**